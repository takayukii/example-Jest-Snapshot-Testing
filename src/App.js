import React from 'react';

function App(props = {}) {
  return (
    <div className="container">
      <div className="jumbotron">
        <h1>{props.greeting}</h1>
      </div>
    </div>
  )
}

export default App
