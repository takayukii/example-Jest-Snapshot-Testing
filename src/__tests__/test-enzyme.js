import React from 'react';
import {mount} from 'enzyme';

import App from '../App';

describe('<App/>', () => {
  it('snapshot2', () => {
    const wrapper = mount(
      <App greeting="Hello enzyme"/>
    );
    expect(wrapper).toMatchSnapshot('Greeting Hello enzyme is displayed?');
  });
});
