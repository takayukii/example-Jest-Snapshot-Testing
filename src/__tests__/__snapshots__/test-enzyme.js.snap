// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting Hello enzyme is displayed? 1`] = `
<App
  greeting="Hello enzyme"
>
  <div
    className="container"
  >
    <div
      className="jumbotron"
    >
      <h1>
        Hello enzyme
      </h1>
    </div>
  </div>
</App>
------------HTML PREVIEW---------------
<App greeting="Hello enzyme">
  <div class="container">
    <div class="jumbotron">
      <h1>
        Hello enzyme
      </h1>
    </div>
  </div>
</App>
`;
